### PR TITLE
Feature/i1902 disabled scanners

### PR DESCRIPTION
**Pull request by yeyisan**
* Updates the API version v1 to v2 for the event status endpoint.
* Implements the new disabled scanner feature.
* Some minor bug fixes.

### CODE REVIEW

File: cmd/scan.go [hunk 1]
```diff
@@ -986,18 +989,24 @@ func (s *Scan) findORCreateProject() (*client.Project, error) {
...
-	case jobFinished:
+	case eventStatusFinished:
 		return "scan finished"
+	case eventStatusFailed:
+		return "scan failed"
 	default:
-		return "unknown scan status"
+		return fmt.Sprintf("unknown status: %d", s)
 	}
 }
```
Comment (canack) [inline cmd/scan.go:1011]: Redundant usage. This method `statusMsg` not used anywhere. Can you check it please?

**Review by canack — APPROVED** [review]
Looks good, some improvements.
We reviewed with @gormelof  (thats means, you got 2 approves 👑 )

File: cmd/scan.go (hunk 1 above)
Comment (yeyisan) [reply cmd/scan.go]: 52c79fc

**Review by canack — APPROVED** [review]
👑

**Review by fleimkeipa — APPROVED** [review]
Nice job ⚡

**Pull request merged**